tests-js/fido2.test.js

```diff
@@ -0,0 +1,35 @@
+import startFido2 from '../src/fido2.js'
+import fetchMock from 'fetch-mock'
+import 'babel-polyfill'
+
+describe('Fido 2', () => {
+
+    test('general test', () => {
```

Comment:
This name is rather general (pun intended), I'd prefer something more specific.

Reply:
Fixed